vue3-webcomponent-wrapper/README.md

```diff
@@ -0,0 +1,27 @@
+# vue3-webcomponent-wrapper
```

Comment:
Could we move this to something like https://github.com/cloudera/hue/tree/master/tools/ and not the root of the project?

Reply:
Thanks, done.